TimeCalculator.csproj

```diff
@@ -94,5 +100,11 @@
   <ItemGroup>
     <None Include="App.config" />
   </ItemGroup>
+  <ItemGroup>
+    <Resource Include="clock.ico" />
+  </ItemGroup>
+  <ItemGroup>
+    <Resource Include="clock.png" />
+  </ItemGroup>
```

Comment:
Wouldn't it be better to put the images in a folder called Images or ressoruces?

Reply:
Same as previous answer